plugins/org.eclipse.xtext.common.types.ui/src/org/eclipse/xtext/common/types/xtext/ui/TypeMatchFilters.java

```diff
@@ -20,17 +21,37 @@ private TypeMatchFilters() {
 		throw new AssertionError("May not be instantiated.");
 	}
 	
+	/**
+	 * @deprecated use {@link #isNonInternal()} instead
+	 */
+	@Deprecated
 	public static ITypesProposalProvider.Filter all() {
 		return new All();
 	}
 	
 	/**
 	 * @since 2.0
+	 * @deprecated use {@link #isNonInternal(int)} instead
 	 */
+	@Deprecated
 	public static ITypesProposalProvider.Filter all(int searchFor) {
 		return new All(searchFor);
 	}
 	
+	/**
+	 * @since 2.9
+	 */
+	public static ITypesProposalProvider.Filter isNonInternal() {
+		return new IsNonInternal();
+	}
+	
+	/**
+	 * @since 2.9
+	 */
+	public static ITypesProposalProvider.Filter isNonInternal(int searchFor) {
```

Comment:
Since there was some confusion about `all`, we should have javadoc on `isNonInternal`. Can we call the method `isNotInternal`?